### ISSUE TITLE
Testing Dataset

### ISSUE THREAD

**Issue by karankumar-07** [hs2573/Diabetic-Retinopathy-Detection#1]
Where is the testing dataset ? (Like y_ train and y_ test). I am new to the CNN. I want to know that without the testing set how you plotted the accuracy ? Usually the dataset contains train, test and validation dataset. Reply ASAP.

**Comment by guptara** [comment]
What is the validation accuracy of your model?

**Comment by vamsi369** [comment]
The testing set would be the same as the training set. But with a fewer pictures and the same structure of folders. In this type of approach when using flow from directory, there is only folder structure where you have a folder for training set and folder for test set, with the same structure of arrangement.
